src/pages/collection/[address].tsx

```diff
@@ -145,17 +190,128 @@ export default function Collection() {
     bg-white border border-gray-200 rounded-lg shadow-md 
     dark:bg-gray-800 dark:border-gray-700 max-w-full max-h-full
     `}>
-      <Line data={data} width={400} height={300} options={options}/>
+
+      <div className="float-right p-2 space-x-2">
+        <select 
+          className="rounded-md p-2 bg-slate-100 hover:bg-slate-200 dark:bg-slate-700 dark:text-slate-300 dark:hover:bg-slate-600 shadow-md"
+          value={selector} 
+          onChange={patchedHandleSelect}
+          >
+          {dataOptions.map((o) => {
+            return <option key={o} value={o}>{o}</option>
+          })}
+        </select>
+        <select 
+          className="rounded-md p-2 bg-slate-100 hover:bg-slate-200 dark:bg-slate-700 dark:text-slate-300 dark:hover:bg-slate-600 shadow-md"
+          value={records} 
+          onChange={(e: any & {target: {value: number}}) => setRecords(e.target.value)}
+          >
+          <option key={7} value={7}>7d</option>
+          <option key={30} value={30}>30d</option>
+          <option key={365} value={365}>1y</option>
```

Comment:
Key is default days. Could add support for hourly and 4hourly charts

Reply:
WIP feature to be added. As of now database does not have capacity or speed to support hourly checkpoints